css/layout.css

```diff
@@ -12,6 +12,33 @@ body {
   margin: auto;
 }
 
+button {
+  padding: 0.6rem;
+  border-radius: 0.4rem;
+  border: 1px solid #7f8cff;
+  flex-basis: auto;
+}
+
+button:hover {
+  color: #fff;
+  border-color: #6070ff;
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+  cursor: pointer;
+}
+
+button:active {
+  background: #2230d2;
+  box-shadow: none;
+}
+
+button:disabled {
+  border: 1px solid #c1c7d0;
+  background: #fff;
+  color: #5e6c84;
+}
+
 .hero * {
   flex-basis: 100%;
   margin-bottom: 0.6rem;
```

Comment:
- remember to create a replacement for this declaration, since, a lot of elements depend on it and it could ruin the layout of the page if deleted.